Docker.md

```diff
@@ -0,0 +1,15 @@
+## development docker-compose.yaml file
+
+### Publishing ports
+
+Ports for services should be explicitly published to the loopback interface, 127.0.0.1 to prevent docker from binding them to 0.0.0.0.
```

Comment:
```suggestion
Ports for services should be explicitly published to the loopback interface, 127.0.0.1 to prevent docker from binding them to 0.0.0.0 (= all interfaces = public to the world).
```